vavr/src/main/java/org/jdbi/v3/vavr/VavrGenericMapUtil.java

```diff
@@ -64,13 +64,39 @@ private static Type resolveMaplikeEntryType(Type maplikeType, Type keyParam, Typ
     }
 
     private static Type resolveMapEntryType(Type keyType, Type valueType) {
-        return resolveMapEntryType(TypeToken.of(keyType), TypeToken.of(valueType));
-    }
+        return new ParameterizedType() {
```

Comment:
I feel like this use case is asking for a `parameterizedType(rawType, typeParamers...)` method in `GenericTypes`